src/cms/api/protected-area/custom/protected-area-status.js

```diff
@@ -270,6 +272,17 @@ const getProtectedAreaStatus = async (ctx) => {
     };
   });
 
+  // filter accessStatus field
+  if (accessStatus) {
+   return payload = payload.filter(
+      (o) => o?.accessStatus?.toLowerCase() == accessStatus.toLowerCase()
+    );
+  } else if (accessStatus_ne) {
+    return payload = payload.filter(
+      (o) => o?.accessStatus?.toLowerCase() != accessStatus_ne.toLowerCase()
+    );
+  }
+
   // Store unfiltered payload into a cached location (unfiltered == no orcs specified)
   if (!ctx.query.orcs) {
```

Comment:
I think we need to update this to include accessStatus and accessStatus_ne (in additon to orcs).

Reply:
yes, I added a case with the both 
please have a look 
thank you